include/cutlass/gemm/kernel/xe_gemm.hpp

```diff
@@ -162,11 +162,28 @@ class GemmUniversal<
     auto m = get<0>(args.problem_shape);
     auto n = get<1>(args.problem_shape);
     auto k = get<2>(args.problem_shape);
+    auto l = get<3>(args.problem_shape);
+    bool is_batch = l > 1;
+    auto check_stride = [is_batch](auto stride, int el_size){
+      auto a = get<0>(stride);
+      auto b = get<1>(stride);
+      auto valid_is_unit = a == _1{} || b == _1{};
+      auto inner = a == _1{} ? b : a;
```

Comment:
```suggestion
      auto valid_is_unit = a == 1 || b == 1;
      auto inner = a == 1 ? b : a;
```

Reply:
Sorry I forgot to reply. I believe this is equivalent without going through some templating. `_1{}` will instantiate a `C<1>` object, then implicitly cast ([defined here](https://github.com/codeplaysoftware/cutlass-sycl/blob/167f160559bb328e59bed7339f261afccc9e3e8d/include/cute/numeric/integral_constant.hpp#L46)) it to the int 1.